source/_posts/2015-12-13-setup-encryption-using-lets-encrypt.markdown

```diff
@@ -46,48 +46,45 @@ First step is to acquire and set up our domain name. For this, go to [DuckDNS],
 
 Let's Encrypt will give you a free 90-day certificate if you pass their domain validation challenge. Domains are validated by having certain data be accessible on your domain for Let's Encrypt ([they describe it better themselves][letsencrypt-technology]).
 
-Assuming that your home is behind a router, the first thing to do is to set up port forwarding from your router to your computer that will run Let's Encrypt. For the Let's Encrypt set up we need to temporary forward ports `80` (http connections) and `443` (https connections). This can be set up by accessing your router admin interface ([Site with port forwarding instructions per router][port-forward]).
+Assuming that your home is behind a router, the first thing to do is to set up port forwarding from your router to your computer that will run Let's Encrypt. For the Let's Encrypt set up we need to forward external port `80` to internal port `80` (http connections). This can be set up by accessing your router admin interface ([Site with port forwarding instructions per router][port-forward]). This port forward must be active whenever you want to request a new certificate from Let's Encrypt, typically every three months. If you normally don't use or have an app that listens to port `80`, it should be safe to leave the port open. This will make renewing certificates easier.
 
-Now you're ready to run Let's Encrypt:
+Now you're ready to install and run Let's Encrypt. The following example will use the [certbot][certbot] client from Let's Encrypt. For a Raspberry Pi 2 on a fresh install of raspbian Jessie:
 
 ```bash
-$ git clone https://github.com/letsencrypt/letsencrypt
-[…]
-$ cd letsencrypt
-$ ./letsencrypt-auto certonly --email your@email.address -d hass-example.duckdns.org
-
-Updating letsencrypt and virtual environment dependencies.......
-Running with virtualenv: sudo /path/letsencrypt/bin/letsencrypt certonly --email your@e-mail.address -d hass-example.duckdns.org
-
-IMPORTANT NOTES:
- - Congratulations! Your certificate and chain have been saved at
-   /etc/letsencrypt/live/hass-example.duckdns.org/fullchain.pem. Your cert
-   will expire on 2016-03-12. To obtain a new version of the
-   certificate in the future, simply run Let's Encrypt again.
- - If like Let's Encrypt, please consider supporting our work by:
-
-   Donating to ISRG / Let's Encrypt:   https://letsencrypt.org/donate
-   Donating to EFF:                    https://eff.org/donate-le
+$ echo 'deb http://ftp.debian.org/debian jessie-backports main' | sudo tee --append /etc/apt/sources.list.d/backports.list
```

Comment:
I'm following the recommended install procedure. Also personally I would never suggest to someone that they should git clone and install, as the first resort, if there is a maintained apt or similar package for their platform. Installing a package from the OS package manager is usually the simplest and most error less way, and updates will be seamless.

I can add a link to the docs where there are instructions for all platforms.

You're call though.